pandas/io/formats/style.py

```diff
@@ -340,6 +343,319 @@ def to_excel(
+
+        .. versionadded:: TODO
+
+        Parameters
+        ----------
+        buf : str, Path, or StringIO-like, optional, default None
+            Buffer to write to. If ``None``, the output is returned as a string.
+        column_format : str, optional
+            The LaTeX column specification placed in location:
+
+            \\begin{tabular}{<column_format>}
+
+            Defaults to 'l' for index and
+            non-numeric data columns, otherwise 'r' (or 'S' if using the
+            {siunitx} package).
+        position : str, optional
+            The LaTeX positional argument (e.g. 'h!') for tables, placed in location:
+
+            \\begin{table}[<position>]
+        position_float : {"centering", "raggedleft", "raggedright"}, optional
+            The LaTeX float command placed in location:
+
+            \\begin{table}[<position>]
+
+            \\<position_float>
+        hrules : bool, default False
+            Set to `True` to add \\toprule, \\midrule and \\bottomrule from the
+            {booktabs} LaTeX package.
+        label : str, optional
+            The LaTeX label included as: \\label{<label>}.
+            This is used with \\ref{<label>} in the main .tex file.
+        caption : str, optional
+            The LaTeX table caption included as: \\caption{<caption>}.
+        sparsify : bool, optional
+            Set to ``False`` to print every item of a hierarchical MultiIndex. Defaults
+            to the pandas ``multi_sparse`` display option.
+        multirow_align : {"c", "t", "b"}
+            If sparsifying hierarchical MultiIndexes whether to align text centrally,
+            at the top or bottom.
+        multicol_align : {"r", "c", "l"}
+            If sparsifying hierarchical MultiIndex columns whether to align text at
+            the left, centrally, or at the right.
+        siunitx : bool, default False
+            Set to ``True`` to structure LaTeX compatible with the {siunitx} package.
+        encoding : str, default "utf-8"
+            Character encoding setting.
+
+        Returns
+        -------
+        str or None
+            If `buf` is None, returns the result as a string. Otherwise returns `None`.
+
+        See Also
+        --------
+        Styler.format: Format the text display value of cells.
+
+        Notes
+        -----
+        **Latex Packages**
+
+        For the following features we recommend the following LaTeX inclusions:
+
+        ===================== ==========================================================
+        Feature               Inclusion
+        ===================== ==========================================================
+        sparse columns        none: included within default {tabular} environment
+        sparse rows           \\usepackage{multirow}
+        hrules                \\usepackage{booktabs}
+        colors                \\usepackage[table]{xcolor}
+        siunitx §             \\usepackage{siunitx}
```

Comment:
are these all fairly standard + stable?

Reply:
yes there are many different options in LaTeX (others will also work) but these were all the most standard and commonly used. Asked the same question of a development partner @moi90 who confirmed.

Reply:
Yes, these are all commonly used and I would consider them stable.